.github/workflows/publish-sdist-wheels.yml

```diff
@@ -82,18 +85,21 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: dist
-          path: dist
+          path: dist/*.tar.gz
+
+  upload_artifacts:
+    name: Upload wheels to PyPI
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
 
-  upload_wheels:
     runs-on: ubuntu-latest
     needs: [build_wheels, build_sdist]
-    name: Upload wheels to PyPI
+
     steps:
       - name: Download artifacts produced during the build_wheels and build_sdist jobs
         uses: actions/download-artifact@v2
         with:
           name: dist
-          path: dist
+          path: dist/
```

Comment:
why the `/`? `rsync`-like syntax stuff?

Reply:
`rsync` behavior drives me crazy

in this case, I don't have a good reason - was a long time since I looked at this

Reply:
at least good to signify that this should be a directory…

Reply:
uhh, yes, let's `recon` my change :)